<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenLIFUTransducerTracking</class>
 <widget class="qMRMLWidget" name="OpenLIFUTransducerTracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>722</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="userAccountBannerPlaceholder" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="userAccountBannerPlaceholderLabel">
        <property name="text">
         <string>Placeholder for a UserAccountBanner widget</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="permissionsWidget" native="true">
     <property name="slicer.openlifu.allowed-roles" stdset="0">
      <stringlist>
       <string>admin</string>
       <string>operator</string>
      </stringlist>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="photoscanGeneratorFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="bottomMargin">
          <number>14</number>
         </property>
         <item>
          <widget class="QLabel" name="photoscanGeneratorFrameTitle">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Photoscan Generation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="photoscanGeneratorButtons">
           <item>
            <widget class="QPushButton" name="startPhotocollectionCaptureButton">
             <property name="toolTip">
              <string>Start a photocollection capture from the 3D Open Water app</string>
             </property>
             <property name="text">
              <string>Start Photocollection Capture</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="importPhotocollectionFromDiskButton">
             <property name="toolTip">
              <string>Import a photocollection from a directory of photos on disk</string>
             </property>
             <property name="text">
              <string>Import Photocollection From Disk</string>
             </property>
             <property name="slicer.openlifu.hide-in-guided-mode" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="startPhotoscanGenerationButton">
           <property name="text">
            <string>Start Photoscan Generation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="photoscanGeneratorProgressBar">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="photoscanGenerationStatusMessage">
           <property name="styleSheet">
            <string notr="true">font-style: italic</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tempWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="algorithmInputWidgetPlaceholder" native="true">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="placeholderLabel">
              <property name="text">
               <string>Placeholder for an OpenLIFUAlgorithmInputWidget</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="previewPhotoscanButton">
        <property name="text">
         <string>Preview Photoscan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runTrackingButton">
        <property name="text">
         <string>Run transducer tracking</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="approvalStatusLabel">
        <property name="text">
         <string>(approval status label)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="approvalWarningLabel">
        <property name="text">
         <string>(approval warnings here)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ModelRenderingOptions">
     <property name="text">
      <string>Model rendering options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QWidget" name="photoscanVisibilitySettings" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="photoscanLabel">
           <property name="text">
            <string>Photoscan visibility:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="photoscanVisibilityCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="opacityLabel">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qMRMLSliderWidget" name="photoscanOpacitySlider">
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
           <property name="quantity">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="skinMeshVisibilitySettings" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="skinMeshLabel">
           <property name="text">
            <string>Skin mesh visibility:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="skinMeshVisibilityCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="opacityLabel_2">
           <property name="text">
            <string>Opacity:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qMRMLSliderWidget" name="skinMeshOpacitySlider">
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
           <property name="quantity">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="viewVirtualFitCheckBox">
        <property name="text">
         <string>View virtual fit transducer position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="workflowControlsPlaceholder" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 0, 128);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qMRMLSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
